config/locales/blacklight.en.yml

```
@@ -15,7 +15,7 @@ en:
       sms: 'SMS'
     search:
       errors:
-        request_error: "Sorry, our search index experienced a problem. Please try again in a moment. If this error persists, please [report the issue](https://libraries.psu.edu/website-feedback) to Libraries Strategic Technology."
+        request_error: "Sorry, our search index experienced a problem. Please try again in a moment. If this error persists, please <a href="https://libraries.psu.edu/website-feedback">report the issue</a> to Libraries Strategic Technology."
```

There are double quotes inside double quotes here, test is failing with syntax error. Maybe sth like below? Of course needs to update references.

```suggestion
        request_error: 
          html: 'Sorry, our search index experienced a problem. Please try again in a moment. If this error persists, please <a href="https://libraries.psu.edu/website-feedback">report the issue</a> to Libraries Strategic Technology.'
```